prow/integ-suite-kind.sh

```diff
@@ -79,6 +80,16 @@ while (( "$#" )); do
   esac
 done
 
+# TODO(landow): remove this, just making sure this works before creating a separate Prow job
+if [[ "${TOPOLOGY}" == "MULTICLUSTER_SINGLE_NETWORK" ]]; then
+  echo "Hacking TOPOLOGY var to be multinetwork"
+  TOPOLOGY="MULTICLUSTER_MULTINETWORK"
+fi
```

Comment:
Currently testing this as if we had a new prow job that sets MULTICLUSTER_MULTINETWORK. It seems like that was the plan. This PR just sets things up so that all existing multicluster tests will succeed with that topology.

Reply:
I think what we'll eventually change the prow setup to have just "multicluster" with one golden topology that tests "everything". We already have 3 clusters... the first 2 will be on the same network and the 3rd will be on a different network. We can then use this configuration for all multicluster tests (and eventually, all Istio tests in general).

In the future, we might consider extending it to 4 clusters. In this case the first 2 would be same network with master/remote, and the second 2 would be on a different network with master/remote. That way we also test dual masters on separate networks.

WDYT?

Reply:
It isn't a lot of work to get it to use all 3: 
https://github.com/stevenctl/istio/pull/1/files

Reply:
I can pull that into this PR, or make it a follow up.
If I make it a follow up, these changes would be pulled in but not run in CI.

Reply:
I'm fine with just doing it here if you like

Reply:
Should be good to go